src/NuGet.Services.AzureSearch/ScoringProfiles/DefaultScoringProfile.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Azure.Search.Models;
+using NuGet.Services.AzureSearch.SearchService;
+
+namespace NuGet.Services.AzureSearch.ScoringProfiles
+{
+    public static class DefaultScoringProfile
+    {
+        public const string Name = "nuget_scoring_profile";
+
+        public static readonly ScoringProfile Instance = new ScoringProfile(
+            Name,
+            textWeights: new TextWeights
+            {
+                Weights = new Dictionary<string, double>
+                {
+                    // Exact match of the package id should be boosted the highest,
+                    // followed by a tokenized match on the package id.
+                    { IndexFields.PackageId, 10 },
+                    { IndexFields.TokenizedPackageId, 5 },
+                }
+            },
+            functions: new List<ScoringFunction>()
+            {
+                // Greatly boost results with high download counts. We score off the log of the download count
+                // with linear interpolation so that the boost slows down at higher download counts. We cannot
+                // use the raw download count with a log interpolation as that would result in a large boosting
+                // range, which would need to be offset by an unmanageably high boosting factor.
+                new MagnitudeScoringFunction(
+                    fieldName: IndexFields.Search.LogBase2DownloadCount,
+                    boost: 100.0,
```

Comment:
Should these values be configurable in a configuration file?

Reply:
Good question. Currently, changing these values requires an index rebuild so I'm leaning more towards keeping these values in code. Please let me know if you feel strongly.